### ISSUE TITLE
Does not create directory

### ISSUE THREAD

**Issue by legacyWork** [eduardogch/gulp-chimp#3]
Error: ENOENT: no such file or directory, mkdir './some/path/to/source/output'

the cause is calling fs.mkdirSync on line 35 in index.js because mkdirSync doess not work with recurisve paths like /source/output

**Comment by jamesrcounts** [comment]
Also, what if logs/screenshots are not desired?  Can we just skip that step?

**Comment by openSourceBugs** [comment]
We do need screenshots and other output.  I may need to consider writing a custom test runner to do this.

**Comment by BriceAubert** [comment]
The main problem is that on line 63 of file `index.js`, the outputFolder name is hardcoded.
Maybe you should consider using a config parameter instead?

```
 62         function (cb) {
 63             createOutputFolder('/source/output');
 64             cb();
 65         },
```

Note that after creating folder `./source/output`, everything worked great :)

**Comment by eduardogch** [comment]
Fix in the new version 1.5.5
